spec/src/main/asciidoc/core/RelatedDocuments.adoc

```diff
@@ -1,137 +1,137 @@
 [[a9849]]
+[bibliography]
 == Related Documents
 
-Enterprise JavaBeans™, version 3.2. (this
+- [[[a9850, 1]]] Enterprise JavaBeans™, version 3.2. (this
 specification) _http://jcp.org/en/jsr/detail?id=3_ 45.
 
-[[a9851]]Java™ Persistence API,
+- [[[a9851, 2]]] Java™ Persistence API,
```

Comment:
We should be replacing the `Java `references with `Jakarta` (in most cases).  Similar thing for `JavaBeans`.  The context surrounding the usage may affect these type of changes, but you'd be safer doing a blanket change and then correcting the individual cases.